Header header
string robot_name
int8 mission_count
int8 task_count
int8[] item_type
int8[] item_count
int8[] obj_type
int8[] obj_count
float64 goal_x
float64 goal_y
string cmd_type
